core/pom.xml

```diff
@@ -353,6 +359,13 @@
                         </configuration>
                     </execution>
                 </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm</artifactId>
+                        <version>7.0</version>
```

Comment:
`7.0` is repeated a lot. Can we make it a property?

Reply:
where's the change ? `${asm.version}`